app/models/goal.py

```diff
@@ -2,4 +2,12 @@
 
 
 class Goal(db.Model):
-    goal_id = db.Column(db.Integer, primary_key=True)
+    goal_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    tasks = db.relationship("Task", back_populates="goal")
+
```

Comment:
Note, you can also create a helper class-method which takes a dictionary as an argument and creates an instance of the class.